mongo/datadog_checks/mongo/collectors/replica.py

```diff
@@ -92,9 +92,18 @@ def _report_replica_set_states(self, members, replset_name):
                     'replset:' + replset_name,
                 ],
             }
+
+            if self.check._config.add_node_tag_to_events:
+                event_payload['host'] = self.hostname
+                event_payload['tags'].append('node:' + node_hostname)
```

Comment:
nit. I think the node tag is a bit too generic. Maybe mongo_node would be better in this case?